Purchases/Public/RCPurchases.m

```diff
@@ -315,6 +311,16 @@ - (instancetype)initWithAppUserID:(nullable NSString *)appUserID
     return self;
 }
 
+- (void)subscribeToAppStateNotifications {
+    [self.notificationCenter addObserver:self
+                                selector:@selector(applicationDidBecomeActive:)
+                                    name:APP_DID_BECOME_ACTIVE_NOTIFICATION_NAME object:nil];
+    [self.notificationCenter addObserver:self
+                                selector:@selector(syncSubscriberAttributesIfNeeded)
```

Comment:
I think to make this clearer, we should do another selector `applicationWillResign` and call `syncSubscriberAttributesIfNeeded` there